ads/getting-started/index.mdx

```diff
@@ -0,0 +1,12 @@
+---
+sidebar_position: 1
+sidebar_label: Getting Started
+---
+
+# Getting started with OptiView Ads
+
+These guides provide the steps required to get started with OptiView Ads (SGAI). They cover the deployment and API integration of the Signaling Service into your existing workflow, as well as the integration of OptiView Ads into your application using various OptiView Player SDKs.
```

Comment:
question: @FabienLavocat I notice that here we're saying "OptiView Player" rather than "THEOplayer". Other places the docs have "THEOplayer". Are we just future proofing here since this is a new file?

Reply:
For the Ads product, I think it is ok to start naming it OptiView Player. Not ready yet in the Player pages.